.gitignore

```diff
@@ -4,3 +4,6 @@ dnzsl-xmldump.xml
 nzsl.dat
 nzsl.db
 picture/*
+nzsl-dictionary-android
+nzsl-dictionary-ios
+*-bak
```

Comment:
Did you mean to add this line 9?

Reply:
I'm sure I added it for a reason, but many of these PRs are changes from quite some time ago, and I can't recall why I added it. I've checked the scripts for mention of bak files, and after not finding any, have removed this line from gitignore.